sdk/go/equinix/fabric/connection.go

```diff
@@ -8,6 +8,7 @@ import (
 	"reflect"
 
```

Comment:
I would imagine goimports would have removed this whitespace

Reply:
yes it was goimports